jvb/src/main/java/org/jitsi/videobridge/rest/JSONDeserializer.java

```diff
@@ -657,7 +657,7 @@ public static RTPHdrExtPacketExtension deserializeHeaderExtension(
         }
         else
         {
-            Long id = (Long)headerExtension.get(RTPHdrExtPacketExtension.ID_ATTR_NAME);
+            String id = (String)headerExtension.get(RTPHdrExtPacketExtension.ID_ATTR_NAME);
```

Comment:
This might break existing software that encodes it as a number. Can we change JSONSerializer instead, since the code was introduced just recently?